.github/workflows/tests.yml

```diff
@@ -91,7 +91,7 @@ jobs:
         run: yarn workspace ${{ matrix.framework }}-example build
 
       - name: Start ${{ matrix.framework }} example
-        run: yarn workspace ${{ matrix.framework }}-example start & npx wait-on -c waitOnConfig.json -t 120000 http-get://localhost:3000/ui/components/authenticator/sign-in-with-username
+        run: yarn workspace ${{ matrix.framework }}-example start & npx wait-on -c waitOnConfig.json -t 20000 http-get://localhost:3000/ui/components/authenticator/sign-in-with-username
```

Comment:
Since all the [framework]-example packages are directly consuming `dist`, we can shorten this to 20 seconds, or less. But I think 20 seconds is short enough to catch anything unexpected during serving without getting intermittent timeouts.